csharp/props/assets.props

```
@@ -0,0 +1,100 @@
+<!--
+  Contains common functionality for all *.Assets.<PLATFORM>.csproj files
+  -->
+<!-- See: https://learn.microsoft.com/en-us/nuget/create-packages/native-files-in-net-packages#projects-targeting-net-framework -->
+<!-- ToDo: Once supporting Net Framework at a base-level, add a hint about non-sdk style projects support being only partial due to build system differences -->
+<Project>
+    <PropertyGroup>
+        <RustDir>$(MSBuildThisFileDirectory)../rust/</RustDir>
+        <TargetFrameworks>netstandard2.0;net8.0;net6.0</TargetFrameworks>
+        <EnforceCodeStyleInBuild>false</EnforceCodeStyleInBuild>
+        <RunAnalyzersDuringBuild>false</RunAnalyzersDuringBuild>
+        <EnableNETAnalyzers>false</EnableNETAnalyzers>
+        <IsPackable>True</IsPackable>
+        <IncludeBuildOutput>false</IncludeBuildOutput>
+        <IncludeSymbols>false</IncludeSymbols>
+        <PackageId>Valkey.Glide.Assets.$(DisplayRuntimeIdentifier)</PackageId>
+        <Configurations>Lint;Release;Debug</Configurations>
+        <RustLibPrefix Condition="'$(OS)' == 'Windows_NT'">lib</RustLibPrefix>
+        <RustLibPrefix Condition="'$(OS)' != 'Windows_NT'"/>
+    </PropertyGroup>
+
+
+    <Target Name="PreClean" BeforeTargets="Clean">
+        <Exec Command="cargo clean --target $(RustTuple)"
+              EnvironmentVariables="CARGO_TERM_COLOR=always"
+              ConsoleToMsBuild="true"
+              WorkingDirectory="$(RustDir)"
+              IgnoreExitCode="true">
+            <Output TaskParameter="ConsoleOutput" PropertyName="OutputOfExec"/>
+        </Exec>
+    </Target>
+
+    <Target Name="PreBuild" BeforeTargets="PreBuildEvent">
+        <Exec Command="echo $(Configuration)" ConsoleToMsBuild="true"/>
+        <Exec Condition="'$(Configuration)' == 'Debug'"
+              Command="cargo build --target $(RustTuple)"
```

```suggestion
              Command="cargo build"
```
I think you can remove that - we do not need cross-compilation. A mac VM will build binary for mac, a linux - for linux and then, once we copy all binaries in one place we can pack and publish.